src/main/java/seedu/address/logic/parser/ListEventsCommandParser.java

```diff
@@ -2,38 +2,37 @@
 
 import static seedu.address.logic.parser.CliSyntax.PREFIX_SORT;
 
-import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.ListEventsCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.SortField;
-
+import seedu.address.model.event.EventSortField;
 
 /**
- * Parses input arguments and creates a new ListCommand object
+ * Parses input arguments and creates and new ListEventsCommand object.
```

Comment:
Small grammar change
```suggestion
 * Parses input arguments and creates a new ListEventsCommand object.
```